src/data/load.js

```diff
@@ -80,7 +83,16 @@ export default function(path, formatter, key, callback) {
       data = validateData(err, parser, data);
       loaded.push(data);
       if (loaded.length - alreadyLoaded === toLoad.length) { // All urls loaded
-        data = formatter ? formatter(loaded.length === 1 ? loaded[0] : loaded) : concat(loaded);
+
+        // Format data
+        data = defaultFormatter(loaded.length === 1 ? loaded[0] : loaded);
```

Comment:
This... is kind of unecessary? This `defaultFormatter` can never be changed by the user, so all if does is grab the first item in the array if there's 1 entry?

Instead of this function, why not just have an `else` in the chain that does this:
```js
else {
  data = loaded.length === 1 ? loaded[0] : loaded;
}
```